src/ports/postgres/modules/utilities/validate_args.py_in

```diff
@@ -365,6 +365,24 @@ def get_cols_and_types(tbl):
     return list(zip(col_names, col_types))
 # -------------------------------------------------------------------------
 
+def get_col_value_and_type(table_name, column_name):
```

Comment:
We could extend this function to handle multiple column_names in addition to a single column_name.

Reply:
That's a good suggestion, but it may not be necessary at the moment. If we see this function getting used more often in the future, let's refactor it out and update it this way.